packages/dd-trace/test/opentracing/tracer.spec.js

```diff
@@ -360,7 +361,14 @@ describe('Tracer', () => {
     it('should handle errors', () => {
       tracer = new Tracer(config)
 
-      expect(() => tracer.inject()).not.to.throw()
+      expect(() => tracer.inject({})).not.to.throw()
+      expect(log.error).to.have.been.calledOnce
+    })
+
+    it('should ignore empty span', () => {
```

Comment:
This test should be moved to the priority sampler test suite.

Reply:
Done